modules/core/src/main/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStore.java

```diff
@@ -1421,6 +1455,18 @@ private void update(@Nullable Entry<? extends K, ? extends V> val,
             this.valStatus = valStatus;
         }
 
+        /**
+         * Added next value that waiting for a previous store operation.
+         *
+         * @param val Value.
+         * @param storeOperation Store operation.
+         */
+        private void addNext(@Nullable Entry<? extends K, ? extends V> val,
```

Comment:
Let's rename this to setNext() because otherwise there is an impression that there is a linked list of store operations.

Reply:
done